rigs/component/profiles/react/config/jest.config.json

```diff
@@ -25,6 +25,8 @@
     }
   },
 
+  "moduleFileExtensions": ["ts", "tsx", "cjs", "js", "jsx", "json", "node"],
```

Comment:
n-b what's a `.node` file extension for?

Reply:
i'm not sure tbh. in the default, it failed because jsx was not present. when it failed, it listed the current config - so i copied it and added jsx